backend/internal/server/server.go

```diff
@@ -28,7 +28,7 @@ func Run() {
 
 	apis.SetupRoutes(app)
 
-	app.Static("/", "/app/static")
+	app.Static("/", "/powerplay/static")
```

Comment:
Pretty sure this is a big enough change it needs to be in its own PR

Reply:
This was already changed by Dr. Christensen. You are okay to push this. It has already been changed in the main project and matches your change.